lawnchair/res/values/strings.xml

```diff
@@ -165,14 +163,16 @@
     <string name="wallpaper">Wallpaper</string>
       <string name="wallpaper_scrolling_label">Scroll Wallpaper</string>
       <string name="wallpaper_depth_effect_label">Wallpaper Depth Effect</string>
-      <string name="wallpaper_depth_effect_description">Zoom into and out of the wallpaper when transitioning between states.</string>
+      <string name="wallpaper_depth_effect_description">Zoom into and out of the wallpaper when transitioning between areas of the launcher.</string>
```

Comment:
> this setting only toggles the wallpaper effect that happens when the notification panel is opened afaik

Well, the effect also happens when you go to the recents, open the app drawer, or swipe to the Google Feed. Basically changed the wording to that since some users are confused on what "states" meant.

I can't think of a different wording, so maybe you could suggest something else that's clear to other users.

Reply:
Hmmm, my problem with the new string is that it says "areas of the launcher" but the effect only happens when you move outside the launcher. Maybe we can just mention recents & notification panel? to avoid confusing the users.

Reply:
I guess I could do that but it'll make the description longer

Also here is the video for that "zoom effect" (recents not included since I don't have QS):
 https://user-images.githubusercontent.com/70206496/187017185-749d4adc-b566-4400-b180-87ff83a21d12.mp4

Reply:
> but the effect only happens when you move outside the launcher

The effect actually happens in the launcher: feed, app drawer, dragging icons (and recents if you have qs). So this wording makes sense. The notification panel also has this effect but that's not controlled by this toggle or the launcher.

Reply:
I guess it's okay then, although I still wish we could find a way to make the string clearer. Since it happens in so many places, I don't think that is possible without way too many lines of text.